### PR TITLE
Added considerations from @nenonix on PR #2586

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding consideration on PR #2586 from @nenonix and @JohanFreelancer9  related to issue #2572 
Please @scanum follow up with this PR.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -84,14 +84,15 @@ Users of Direct Routing must have the following licenses assigned in Office 365:
...
-> [!NOTE]
-> When you are escalating from a 1:1 call the audio conferencing license is required
 
 > [!IMPORTANT]
 >  In the case that you would like to add external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number, the audio conferencing license is *required*.
 
 > [!NOTE]
-> The E5 license includes both Phone System and Audio Conferencing.   
+> The audio conferencing license is *required* to:
+> - Escalate from 1:1 call to a group call
```
Comment (nenonix) [inline Teams/direct-routing-plan.md]:
```suggestion
> - Escalate from 1:1 call to a group call.
```

**Comment by LolaJacobsen** [comment]
@CarolynRowe - for review

**Comment by j0rt3g4** [comment]
@LolaJacobsen Done reviewing :)

**Pull request merged**